packages/core/src/services/ai/providers/models/index.ts

```diff
@@ -1,5 +1,3 @@
-import { omit } from 'lodash-es'
```

Comment:
Why this? We're already using lodash in other places

Reply:
I don't know why but it fails to build the webapp (that same omit and code was in a component). I think this is the simplest way to avoid locking the CD pipeline for now.

```text
../../packages/core/src/services/ai/providers/models/index.ts
Dynamic Code Evaluation (e. g. 'eval', 'new Function', 'WebAssembly.compile') not allowed in Edge Runtime 
Learn More: https://nextjs.org/docs/messages/edge-dynamic-code-evaluation

The error was caused by importing 'lodash-es/lodash.js' in '../../packages/core/src/services/ai/providers/models/index.ts'.
```